visidata/guides/ColumnsGuide.md

```diff
@@ -0,0 +1,58 @@
+# Rename, hide and resize columns
+
+How to rename, hide, and resize columns to focus on key data. These actions
+help organize the sheet, making it easier to prioritize information
+and reduce distractions. 
+
+# Renaming columns
+
+- {help.commands.rename_col}
+
+## Renaming one or more columns using data 
+
+Rename one or more columns using data from the sheet:
+
+- {help.commands.rename_col_selected}
+- {help.commands.rename_cols_row}
+- {help.commands.rename_cols_selected}
+
+## Clean column names
+
+Rename columns to be valid python identifiers that can be used in column
+expressions.
+
+- {help.commands.normalize_col_names}
+- {help.commands.clean_names}
+
+# Hiding and Unhiding columns
+
+Clean up the sheet by hiding unwanted columns.  Hidden columns are still present, just not visible.
```

Comment:
In the upcoming version, hidden columns are 1 character columns on the right, which can be unhidden with `_`.

Reply:
interesting! I see how that would make it much easier to understand what is going on.